content/en/docs/languages/java/instrumentation.md

```diff
@@ -33,6 +33,13 @@ _natively_ or you can make use of
 
 {{% /alert %}}
 
+## Prerequisites
```

Comment:
as discussed below, can we move those prerequisites to within the sample app setup, and make it much more clearer that this is for that sample app